timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -155,6 +159,10 @@ types:
       ConjureLockTokenV2:
         alias: uuid
         safety: safe
+      ConjureLeasedLockToken:
+        fields:
+          lockToken: LockToken
```

Comment:
Do you know why you used LockToken? vs ConjureLockToken? vs ConjureLockTokenV2? :P

Reply:
No. I check most of them and they all wrap a request id. I tracked the calls for locking the immutable timestamp and for unlocking it from snapshot transaction.

All that mattered is that you held to the original uuid. If there is a deeper reason, I can switch! (this feels probably like migrating things)

Reply:
Maybe just check with Jeremy, he might have some historical context and overall sense of direction.

Reply:
For this one:
- To double check, but I don't think we use `LockToken` directly in the definition any more so we should probably kill it, actually.
- `ConjureLockToken` is the "main" one; you'll see that `ConjureLockTokenV2` is only used in the Refresh and Unlock endpoints, and it's basically a UUID alias vs an object containing only a UUID. This was done as part of my hackweek project a few years back to save money on timelock cross-AZ RPCs. See #6160 
- I know that we currently use locking to implement this, but that's probably an implementation detail - so maybe we should just have a `LeaseGuarantee` or something like that that's a UUID alias?